digitron.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    
+<link rel="stylesheet" type="text/css" href="stil.css">
+<script type="text/javascript" src="racunaljka.js"> </script>
+
+</head>
+
+<body>
+    
+<form> 
+    
+<input type='text' size='15' id='izvestaj' > <br/><br/>
+<input type="button" value='1' onclick='brojJedan(); return seeAll()'> <input type="button" value="2" onclick="brojDva(); return seeAll()">
```

Comment:
HTML should be used as view (representation of data), while JavaScript should be used as a controller (usage and modification of data). This is a bad approach, since you are adding functionality in a view. I am referring to:
```
onclick='brojJedan(); return seeAll()'
```
This is something which should be defined inside the controller.

P.S. Investigate MVC - Model View Controller, a standard approach when working with UI.